_posts/2020-04-13-squeezing-the-most-out-of-your-pebble.md

```diff
@@ -0,0 +1,132 @@
+
+Let's start with the easy questions first...
+
+<!--more-->
+
+
+# What is RWS / Why do I need it?
+RWS stands for Rebble Web Services, and is what most people mean when they say 'Rebble' (though Rebble encompasses more than RWS, I'll get to that later). RWS replaces the old Pebble servers, but does not replace the app - instead, during setup, you're telling the original Pebble app to point to Rebble servers. Without RWS you'll have to sideload Pebble apps, and won't get any of the other server-dependant functions. RWS provides two tiers: Free and Paid. Here's what you get with each:
+#### Free Tier:
+   - Access to the Rebble app store
+   - Timeline Server Support (3 hour sync period)
+
+#### Paid Tier:
+   - Access to the Rebble app store
+   - Timeline Server Support (30 minute sync period)
+   - Voice Dictation (note that iOS SMS reply doesn't work)
+   - Weather API (For the system weather app and timeline pins, watchfaces do not need this as they handle it themselves)
+   - You fund the free tier and other Rebble work
+
+# How do I get setup with RWS?
+Follow the guide at [rebble.io/howto](https://rebble.io/howto). Do not use any 'Rebble' apps from the appstore, as they aren't legitimate and probably don't work. If you run into any issues, check the 'common issues' section under the howto, or ask in #rebble-help on [Discord](https://rebble.io/Discord).
+
+# What doesn't work anymore?
+Since the Pebble shutdown in 2016 some small features don't work anymore, namely:
+
+   - Timeline based apps that use '[topics](https://developer.rebble.io/developer.Pebble.com/guides/Pebble-timeline/timeline-js/index.html#subscribe-to-a-topic)', which allow lots of users to subscribe to a single source of timeline pins. But this is a small percentage of timeline apps, the rest still work. Note that your calendar pins are special, and do not require the timeline server to work.
+   - Reply to SMS on iOS - Pebble used clever carrier-specific workarounds for iOS' locked down SMS behaviour in a way that Rebble can't easily replicate.
+   - Apps that rely on servers no longer available - Some apps serve their settings page from web servers, and if those servers don't exist anymore you can't setup the apps. [It's not always a total loss though](https://old.Reddit.com/r/Pebble/comments/eccrn7/getting_an_error_when_i_try_to_access_the/).
+
+# So what *can* I do with my Pebble in 2020?
+The sky is the limit! Most of the apps and watchfaces on the appstore still work, and more are still being published - Like swanswanswanswansosoft's [Covid-19 tracker](https://apps.rebble.io/en_US/application/5e65d05a3dd3108f341c08ec).
+
+### Apps
+Here are some great apps that still work and are useful daily:   
+
+App | Description | Platform   
+:--|:--|:--|:--|:--   
+[Checklist](https://apps.rebble.io/en_US/application/5620e876768e7ada4e00007a) | A checklist of items, for shopping etc | Both
```

Comment:
I am the worst person ever, but can we get a `.` after both uses of `etc`? :heavy_heart_exclamation: